tools/opt/opt.cpp

```diff
@@ -361,10 +364,8 @@ int __cdecl main(int argc, char **argv) {
   //initializeDwarfEHPreparePass(Registry); // HLSL Change: remove EH passes
   //initializeSjLjEHPreparePass(Registry);  // HLSL Change: remove EH passes
   // HLSL Change Starts
-  initializeReducibilityAnalysisPass(Registry);
```

Comment:
This pass is actually initialized a few lines above as well. It was redundant then and redundant now, but we might as well remove it.

Reply:
Good catch.
Fixed.